src/pudl/transform/ferc1.py

```diff
@@ -963,7 +966,7 @@ def reconcile_table_calculations(
         and ``rel_diff``.
     """
     # If we don't have this value, we aren't doing any calculation checking:
-    if params.column_to_check is None or calculation_components.empty:
+    if params.column_to_check is None:
```

Comment:
previously the `depreciation_by_function` table was entirely getting skipped over because it has no calc components prior to inferring the total to sub-dim calcs via `_add_intra_table_calculation_dimensions`